modules/diskq/dqtool.c

```diff
@@ -506,6 +525,70 @@ dqtool_relocate(int argc, char *argv[])
   return 0;
 }
 
+static gboolean
+_assign_validate_options(void)
+{
+  return _validate_persist_file_path(persist_file_path);
+}
+
+static void
+_assign_print_help(void)
+{
+  fprintf(stderr, "example:"
+          "   bin/dqtool assign -p var/syslog-ng.persist -n \"afsocket_dd_qfile(stream,localhost:15554)\""
+          "                    /tmp/syslog-ng-dq/syslog-ng-00000.rqf\n\n"
```

Comment:
The indenting seems a little odd here. It looks like this in action:
```
$ ./dqtool assign --example
example:   bin/dqtool assign -p var/syslog-ng.persist -n "afsocket_dd_qfile(stream,localhost:15554)"                    /tmp/syslog-ng-dq/syslog-ng-00000.rqf
```
Apart from this formatting inconvenience, it is an approve from me.